Userland/Libraries/LibWeb/CSS/StyleValues/ImageStyleValue.h

```diff
@@ -26,6 +28,12 @@ class ImageStyleValue final
     }
     virtual ~ImageStyleValue() override = default;
 
+    void visit_edges(JS::Cell::Visitor& visitor) const
+    {
+        // FIXME: visit_edges in non-GC allocated classes is confusing pattern.
```

Comment:
:thinking: Maybe it wouldn't be crazy to GC-allocate StyleValues. Though, ImageStyleValue is the only one that seems to need it.

Reply:
Yeah seems uncrazy to me. Anything whose lifetime interacts with GC lifetimes is generally easier to just let the GC figure out.

Reply:
:yakplus:

Reply:
sure, I am going to make StyleValue GC-allocated

Reply:
ok, I started converting StyleValue to be GC-allocated and then found I also have to convert `CSS::Size` and `LengthPercentage` to be GC-allocated or use `JS::Handle`.

so I would prefer to only fix leaking in a bit ugly way here.

Reply:
Oof, yeah that sounds like something we don't want to do.